### PR TITLE
Added check to see if ssh key was already gone before trying to delete it

### PR DESCRIPTION
**Pull request by karafelix**
[Finishes #106029472]

### CODE REVIEW

File: test_helpers/test_helpers.go [hunk 1]
```diff
@@ -438,21 +442,36 @@ func WaitForVirtualGuestToHaveNoActiveTransactions(virtualGuestId int) {
...
+	accountService, err := CreateAccountService()
+	Expect(err).ToNot(HaveOccurred())
+	sshKeys, err := accountService.GetSshKeys()
+	Expect(err).ToNot(HaveOccurred())
+
+	present := false
+	for _, sshKey := range sshKeys {
+		if sshKey.Id == sshKeyId {
+			present = true
+		}
```
Comment (maximilien) [inline test_helpers/test_helpers.go]: why don't you break here? what's the point of continuing the search once you found the key?

**Comment by maximilien** [comment]
Looks good. Please address my comment and make sure to `go fmt` all sources and update the PR. I'll merge it. Thanks.

**Comment by karafelix** [comment]
That would be because I copied it from WaitForDeletedSshKeyToNoLongerBePresent and WaitForCreatedSshKeyToBePresent, both of which didn't do that either, and didn't think about it.

I've updated all three to now break out once they find the answer they want.

**Comment by maximilien** [comment]
Cool, cheers :beers:

**Pull request merged**